src/main/java/uk/org/okapibarcode/output/PostScriptRenderer.java

```diff
@@ -104,6 +135,25 @@ public void render(Symbol symbol) throws IOException {
             writer.append("/TR { newpath 4 1 roll exch moveto 1 index 0 rlineto 0 exch rlineto neg 0 rlineto closepath fill } bind def\n");
             writer.append("/TE { pop pop } bind def\n");
 
+            // Set orientation
+            switch (rotation) {
+                case 90:
+                case 270:
+                    writer.append("gsave\n");
+                    writer.append(height/2).append(" ").append(width/2).append(" translate\n");
```

Comment:
I think all of these `height/2` and `width/2` calculations will be off by 0.5 if the size is an odd number (i.e. it gets rounded and we lose the 0.5). I think we should be OK if we divide by `2d` instead, so that the result of the division is a double rather than an int. We also tend to add spaces on both sides of the `/`, so let's address both with the following changes: `height/2` -> `height / 2d` and `width/2` -> `width / 2d`. Let me know if that makes sense? (Ditto in the other two renderers where we divide by an int for the new angle or translation calculations.)